addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/discovery/EvohomeDiscoveryService.java

```diff
@@ -94,48 +87,46 @@ private void discoverHeatingZones(int locationId, TemperatureControlSystem heati
             properties.put(EvohomeBindingConstants.ZONE_MODEL_TYPE, modelType);
             addDiscoveredThing(thingUID, properties, zoneName);
         }
-        // TODO Auto-generated method stub
-
     }
 
-    private void discoverGateway(ControlSystem controlSystem) {
+    private void discoverDisplay(ControlSystem controlSystem) {
         String name = controlSystem.getName();
-        ThingUID thingUID = findThingUID(THING_TYPE_EVOHOME_DISPLAY.getId(), name);
+        ThingUID thingUID = findThingUID(EvohomeBindingConstants.THING_TYPE_EVOHOME_DISPLAY.getId(), name);
         Map<String, Object> properties = new HashMap<>();
-        properties.put(EvohomeBindingConstants.LOCATION_NAME, name);
-        properties.put(EvohomeBindingConstants.LOCATION_ID, controlSystem.getId());
+        properties.put(EvohomeBindingConstants.DEVICE_NAME, name);
+        properties.put(EvohomeBindingConstants.DEVICE_ID, controlSystem.getId());
```

Comment:
I think so yeah? I'll add that as a TODO, and check if that is possible. Maybe you could share your gateway with me so I can check that (jk ;-) )

For now I would say we just support one display